design-toolkit/components/src/components/tabs/index.tsx

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { cn } from '@/lib/utils';
+import { cva } from 'cva';
+import type React from 'react';
+import {
+  Tabs as AriaTabs,
+  TabList as AriaTabList,
+  Tab as AriaTab,
+  TabPanel as AriaTabPanel,
+  type TabsProps as AriaTabsProps,
+  type TabListProps as AriaTabListProps,
+  type TabProps as AriaTabProps,
+  type TabPanelProps as AriaTabPanelProps,
+} from 'react-aria-components';
+
+/**
+ * This is a tabs section.
+ */
+export interface TabsProps extends AriaTabsProps {}
+
+export function Tabs({
+  children,
+  className,
+  orientation = 'horizontal',
+  isDisabled = false,
+}: TabsProps) {
+  return (
+    <AriaTabs
+      orientation={orientation}
```

Comment:
```suggestion
  isDisabled = false,
  ...rest
}: TabsProps) {
  return (
    <AriaTabs
    ...rest
      orientation={orientation}
```
You'll want to spread the rest of the supported props into the component.